### PR TITLE
Remove the card "Visit the theme catalog" from Theme config page

### PR DESCRIPTION
**Pull request by intraordinaire**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Remove the card "Visit the theme catalog" in the theme configuration page in back office.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | YES
| Deprecations?     | no
| Fixed ticket?     | Par of #24639 
| How to test?      | The card "Explore theme" in Theme & Logo page have to be deleted.

📓 BC Break : 
 - Removed template
   - `@PrestaShop/Admin/Improve/Design/Theme/Blocks/Partials/theme_catalog_card.html.twig`

image
image

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26917)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Theme/index.html.twig [hunk 1]
```diff
@@ -138,23 +138,6 @@
           {% include '@PrestaShop/Admin/Improve/Design/Theme/Blocks/delete_theme_modal.html.twig' %}
         {% endif %}
 
-        {% embed '@PrestaShop/Admin/Improve/Design/Theme/Blocks/Partials/theme_catalog_card.html.twig' %}
```
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Theme/index.html.twig:141]: Please remove the template too

File: src/PrestaShopBundle/Controller/Admin/Improve/Design/ThemeController.php [hunk 2]
```diff
@@ -83,29 +83,12 @@ class ThemeController extends AbstractAdminController
...
-        $languageAddons = in_array(strtolower($isoCode), $languagesAddons) ? strtolower($isoCode) : 'en';
-
-        $themeCatalogUrl = sprintf(
-            '%s?%s',
-            'https://addons.prestashop.com/' . $languageAddons . '/3-templates-prestashop',
-            http_build_query([
-                'utm_source' => 'back-office',
-                'utm_medium' => 'theme-button',
-                'utm_campaign' => 'back-office-' . $isoCode,
-                'utm_content' => $isHostMode ? 'cloud' : 'download',
```
Comment (matks) [inline src/PrestaShopBundle/Controller/Admin/Improve/Design/ThemeController.php:98]: The past is never dead

**Review by Progi1984 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by florine2623** [comment]
Hello @intraordinaire ,

Looks good to me, thank you !

**Comment by Progi1984** [comment]
Thanks @intraordinaire & @florine2623

**Pull request merged**